src/functions/template-tags/html.php

```diff
@@ -79,9 +79,9 @@ function tribe_required( $required, $echo = true ) {
 
 		if ( ! $echo ) {
 			return $required_string;
-		} else {
-			echo $required_string;
 		}
+
+		echo $required_string;
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$required_string'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$required_string'.

<sub>hash: ef2739b5a7ce84b598c4be8a2321c466</sub>